wallet/chainntfns.go

```diff
@@ -305,6 +307,21 @@ func (w *Wallet) addRelevantTx(dbtx walletdb.ReadWriteTx, rec *wtxmgr.TxRecord,
 		for _, addr := range addrs {
 			ma, err := w.Manager.Address(addrmgrNs, addr)
 			if err == nil {
```

Comment:
nit: it might make sense to simplify how the error checks are done, and avoid one identation

```
ma, err := w.Manager.Address(addrmgrNs, addr)

// Missing addresses are skipped.  
if waddrmgr.IsError(err, waddrmgr.ErrAddressNotFound) {
        continue
}
// Other errors should be propagated.
if err != nil {
        return err
}

// Prevent addresses from non-default scopes to
// be detected here. We don't watch funds sent
// to non-default scopes in other places either,
// so detecting them here would mean we'd also
// not properly detect them as spent later.
scopedManager, _, err := w.Manager.AddrAccount(
...
```

Reply:
Good idea, done.